backend/sozisel/test/sozisel_web/schema/events/quiz_mutations_test.exs

```diff
@@ -0,0 +1,267 @@
+defmodule SoziselWeb.Schema.QuizMutationsTest do
+  use SoziselWeb.AbsintheCase
+
+  import Sozisel.Factory
+
+  alias Sozisel.Model.Events
+
+  @create_quiz """
+  mutation CreateQuiz($input: CreateQuizInput!) {
+    createQuiz(input: $input) {
+      id
+      name
+      start_minute
+      event_structure {
+        ... on Quiz {
+          duration_time_sec
+          target_percentage_of_participants
+          tracking_mode
+          quiz_questions {
+              question
+              answers
+              correct_answers
+          }
+        }
+      }
+      session_template_id
```

Comment:
We should probably return session and then catch its id instead of providing `sessionTemplateId`
Should be something like
```
...
  sessionTemplate {
    id
  }
```